### PR TITLE
DE616306 readme upate

### PR DESCRIPTION
**Pull request by zeibura**
clarify breadcrumbs and snippets

### CODE REVIEW

**Comment by zeibura** [comment]
also clarify what semantic analysis is supported

**Comment by zeibura** [comment]
Two further changes
- "open a program or project" - "open a file or folder". Project isn't VS Code terminology - possibly a holdover from Eclipse Che days
- clarify that remote copybooks can't be edited and saved back to mainframe, recommend using Zowe Explorer

File: README.md [hunk 1]
```diff
@@ -72,21 +72,21 @@ The extension enables syntax highlighting for COBOL code.
...
+1. Press **F1** to open the command palette.
 2. Type **Insert Snippet** and press enter.
 3. Select the snippet that you want to insert.
 
 ![Code snippets](/docs/images/Snippet.gif)
 
 You can also insert a code snippet by typing the name of the snippet in your code and clicking on the autocomplete text.
 
-The COBOL Language Support extension also supports user snippets. Add your custom snippets to the `COBOL.json` file in your user snippets folder.
+The COBOL Language Support extension also supports user snippets. Add your custom snippets to the `COBOL.json` file in your user snippets folder. To access your user snippets file, select **File** - **Preferences** - **Configure Snippets** and select `COBOL.json` from the list.
```
Comment (VitGottwald) [inline README.md]: I a unable to find File -> Preferences. On macOS it is under Code -> Settings -> Configure snippets.
How about using the command palette instead: F1 -> Snippets: Configure Snippets ?
image

Comment (VitGottwald) [reply README.md]: That sholuld be the same on all platforms.

Comment (zeibura) [reply README.md]: Done

**Review by VitGottwald — APPROVED** [review]

**Pull request merged**